plugins/modules/template_workflow_manager.py

```diff
@@ -3729,8 +3728,7 @@ def get_diff_deleted(self, config):
             self.set_operation_result(
                 "failed", False, self.msg, "ERROR").check_return_status()
 
-        self.msg = "Successfully completed delete state execution"
-        self.status = "success"
+        self.log("Successfully completed delete state execution", "DEBUG")
```

Comment:
Can we also print with template name?

Reply:
CAn we say
self.log("Successfully completed the delete operation.", "DEBUG")
With template name ?

Reply:
done.